docs/compilation/multi_precision.md

```diff
@@ -44,10 +48,10 @@ After this graph processing step, the graph would look like:
 %2 = add(%0, %1)        # EncryptedScalar<uint5>
 return %2
 ```
+## Encoding flexibility with Table Lookup
```

Comment:
```
## Encoding flexibility with table lookup
```

Reply:
I think it's better to keep the same writing of "table lookup" across docs. As it's a special term that we use quite often, I propose to keep the capitalization everywhere as "**Table Lookup (TLU)**"

Reply:
ok